### ISSUE TITLE
Program Styling Updates

### ISSUE THREAD

**Issue by charlieartist** [Drupal4Gov/Drupal-GovCon-2016#87]
Fix the double field display issues in the Sessions view, as well as updated and improved styling for UX.

**Comment by timwood** [comment]
Can you explain or show an example of this issue?

**Comment by charlieartist** [comment]
Yes--look here: https://www.drupalgovcon.org/drupal-govcon-2016/schedule

![screen shot 2016-05-27 at 10 29 02 pm](https://cloud.githubusercontent.com/assets/3085730/15624629/9f9500a0-245a-11e6-81c8-5f99ef953846.png)
